examples/jupiter-swap/src/app/layout.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const metadata = {
+  title: 'Next.js',
```

Comment:
should you change this to something else ?

Reply:
Removed, not needed for frames